custom_components/deutschebahn/sensor.py

```diff
@@ -62,6 +65,20 @@ def __init__(self, config, hass: HomeAssistantType):
         self.start = config[CONF_START]
         self.goal = config[CONF_DESTINATION]
         self.offset = timedelta(minutes=config[CONF_OFFSET])
+        self.max_connections: int = config[CONF_MAX_CONNECTIONS]
+        self.select_any_product: bool = any(
+            [
+                p.strip().upper() in ["*", "ALL", "ANY"]
+                for p in config[CONF_SELECTED_PRODUCTS]
```

Comment:
Please provide an updated Readme what are valid options for this config option. For example ICE, and so on. 

Also a check if the user has input a valid option would be great, since inputting anything not valid will result in the sensor being created with "unknown" and most likel the user then creating an issue here. 
For example this would also include typos, like sbahn vs. s-bahn and so on.

Reply:
I added a UI selection list for that. And I reversed the semantic on that one (from "selected products" to "ignored products") to accommodate that i currently don't know all the abbreviations which are available in the DB universe.